_inc/client/components/plans/plan-price/index.jsx

```diff
@@ -14,72 +14,97 @@ import { getCurrencyObject } from '@automattic/format-currency';
 import './style.scss';
 
 export class PlanPrice extends Component {
-	render() {
-		const { currencyCode, rawPrice, original, discounted, className } = this.props;
+	getPriceRange() {
+		const { currencyCode, rawPrice } = this.props;
 
-		if ( ! currencyCode || ! rawPrice ) {
-			return null;
+		// "Normalize" the input price or price range
+		let rawPriceRange;
+		if ( Array.isArray( rawPrice ) ) {
+			const positivePrices = rawPrice.filter( price => price >= 0 );
+			// First array entry is lowest price, second is highest price.
+			rawPriceRange = [ Math.min( ...positivePrices ), Math.max( ...positivePrices ) ];
+		} else {
+			rawPriceRange = [ rawPrice ];
```

Comment:
Updated the code in af6efc72ccedc4c7dddfa0b1322c208427b9c268 to add some additional sanitization, as the code was assuming the first array entry must be the lowest price, and second highest. Let me know if I should make a seperate PR for this, @delawski :)

Reply:
Thank you for this update! Let's keep it in this PR - I think it's totally justified to have it here 🙂